src/forklift/plugin.py

```diff
@@ -13,21 +13,20 @@
 
 class ScheduledUpdateBase(object):
 
-    def __init__(self):
-        #: the logging module to keep track of the plugin
-        self.log = logging.getLogger(settings.LOGGER)
-        #: the duration in hours that marks a plugin as dirty
-        self.expires_in_hours = 24
-        #: the parent path to where the source data is
-        self.source_directory = 'C:\\MapData\\'
-        #: the path to where the data in dependencies resides
-        self.source_location = 'SGID10.sde'
-        #: the parent path to where the output will be created
-        self.output_directory = 'C:\\MapData\\'
-        #: the file geodatabase where data will be inserted
-        self.output_gdb_name = 'SGID10.gdb'
-        #: the table names for all dependent data for an application
-        self.dependencies = []
+    #: the duration in hours that marks a plugin as dirty
+    expires_in_hours = 24
+    #: the table names for all dependent data for an application
+    dependencies = []
```

Comment:
i think arrays will be shared across all instances of the class like an array in javascript. That is why I chose to move them all into the init. You might want to validate that concern.